File: app/src/main/java/com/ytempest/wanandroid/base/view/IView.java
package com.ytempest.wanandroid.base.view;

import androidx.annotation.StringRes;

/**
 * @author heqidu
 * @since 2020/6/28
 */
public interface IView {
    void showToast(String msg);

    void showToast(@StringRes int textId);

    void showLoading();

    void stopLoading();
}

File: app/src/main/java/com/ytempest/wanandroid/base/view/IView.kt
package com.ytempest.wanandroid.base.view

import androidx.annotation.StringRes

/**
 * @author heqidu
 * @since 21-2-7
 */
interface IView {

    fun showToast(msg: String?)

    fun showToast(@StringRes textId: Int)

    fun showLoading()

    fun stopLoading()
}